src/main/java/cs/ualberta/cmput402/tictactoe/board/Board.java

```diff
@@ -19,11 +19,11 @@ public Board(){
         currentPlayer = Player.X;
     }
 
-    private void initBoard(){
+    public void initBoard(){
```

Comment:
It's better to keep this method private.
```suggestion
    private void initBoard(){
```

Reply:
I changed this method to public because I used the function outside of the class.